pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -35575,36 +35374,6 @@ self: {
      }) {};
 
   "attoparsec" = callPackage
-    ({ mkDerivation, array, base, bytestring, case-insensitive
-     , containers, criterion, deepseq, directory, filepath, ghc-prim
-     , http-types, parsec, QuickCheck, quickcheck-unicode, scientific
-     , tasty, tasty-quickcheck, text, transformers, unordered-containers
-     , vector
-     }:
-     mkDerivation {
-       pname = "attoparsec";
-       version = "0.13.2.5";
-       sha256 = "0vv88m5m7ynjrg114psp4j4s69f1a5va3bvn293vymqrma7g7q11";
-       revision = "2";
-       editedCabalFile = "1syz7hf2h8c8vwqniymblnyciwqh4hvflfp1h0ywg7ad8q4gqr0v";
-       libraryHaskellDepends = [
-         array base bytestring containers deepseq ghc-prim scientific text
-         transformers
-       ];
-       testHaskellDepends = [
-         array base bytestring deepseq QuickCheck quickcheck-unicode
-         scientific tasty tasty-quickcheck text transformers vector
-       ];
-       benchmarkHaskellDepends = [
-         array base bytestring case-insensitive containers criterion deepseq
-         directory filepath ghc-prim http-types parsec scientific text
-         transformers unordered-containers vector
-       ];
-       description = "Fast combinator parsing for bytestrings and text";
-       license = lib.licenses.bsd3;
-     }) {};
-
-  "attoparsec_0_14_4" = callPackage
```

Comment:
I think we should consider keeping old `attoparsec` version as `attoparsec_0_13` here. At least `elmPackages.elm` needs this version (via snap-core) but there might be more packages like that. Either we can keep it in haskellPackages set or we can do the override in elmPackages

Reply:
Yes, this is easily done by adding an `attoparsec == 0.13.*` entry in `configuration-hackage2nix/main.yaml`'s `extra-packages` section and running `maintainers/script/haskell/regenerate-hackage-packages.sh`.

Reply:
@turboMaCk downgraded attoparsec in 376a511e0997247b8d3a8f46b63a83d6444bf7d2.